platform/public_api/access_control/index.json

```diff
@@ -0,0 +1,470 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Access Control Api",
+    "version": "2.0"
+  },
+  "parameters": {
+    "Entity_Id": {
+      "in": "path",
+      "name": "entityId",
+      "type": "string",
+      "format": "uuid",
+      "required": true,
+      "description": "UUIDv4 identifier of given entity"
+    },
+    "Entity": {
+      "in": "path",
+      "name": "entity",
+      "description": "entity",
+      "type": "string",
+      "required": true,
+      "enum": [
+        "ppms/app",
+        "ppms/user"
+      ]
+    }
+  },
+  "tags": [
+    {
+      "name": "AccessControl v2"
+    }
+  ],
+  "definitions": {
+    "JsonApi_Id": {
+      "type": "string",
+      "format": "uuid",
+      "description": "UUIDv4 string",
```

Comment:
It describes format instead of what this param is used for / what it is. For format description use `format` attribute.

Reply:
actually its a definition for JsonAPI object, so sometimes its a `entity ID`, sometimes `User ID`. @jzyinq do you agree?

Reply:
That's correct. Since it's generally returns same id that was queries maybe we can name this something like `Id of requested resource` ?

Reply:
`Resource ID`